client/src/components/application/DotsStepper/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import MobileStepper from '@material-ui/core/MobileStepper';
+import PropTypes from 'prop-types';
+import dotstyle from './style';
+import regestration from './regestrationSteps.json';
+import Typography from '../../common/Typography';
+import Card from '../../common/card';
+import Button from '../../common/Button';
+
+const { primaryHeader, Info, stepHeader, steps } = regestration;
+function DotStepper({ inSteps }) {
+  const classes = dotstyle();
+  const [activeStep, setActiveStep] = React.useState(0);
+  const maxSteps = inSteps.length;
+
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
+  };
+
+  const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  };
+
+  return (
+    <Card
+      content={
+        <div>
+          <div className={classes.content}>
+            <div>
+              <Typography variant="h4" color="primary">
+                {primaryHeader}
+              </Typography>
+              <Typography variant="subtitle1" color="textPrimary">
+                {Info}
+              </Typography>
+            </div>
+            <div>
+              <Typography variant="body2" color="secondary">
```

Comment:
change fontsize for body1 in Typography component to be : 
```js
body: {
      fontFamily: 'Raleway',
      fontSize: 13,
      fontWeight: 700,
    },
```
Also try to reduce spaces